tests/phpunit/Unit/Graph/GraphTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SRF\Tests\Unit\Formats;
+
+use SMW\Test\QueryPrinterRegistryTestCase;
+use SRF\Graph\GraphNode;
+
+
+/**
+ * Tests for the SRF\Graph class.
+ *
+ * @file
+ * @since 1.8
+ *
+ * @ingroup SemanticResultFormats
+ * @ingroup Test
+ *
+ * @group SRF
+ * @group SMWExtension
+ * @group ResultPrinters
+ *
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class GraphTest extends QueryPrinterRegistryTestCase {
+
+	/**
+	 * @see QueryPrinterRegistryTestCase::getFormats
+	 *
+	 * @since 1.8
+	 *
+	 * @return array
+	 */
+	public function getFormats() {
+		return [ 'graph' ];
+	}
+
+	/**
+	 * @see QueryPrinterRegistryTestCase::getClass
+	 *
+	 * @since 1.8
+	 *
+	 * @return string
+	 */
+	public function getClass() {
+		return 'SRF\Graph\GraphPrinter';
+	}
+
+	/**
+	 * Testing class GraphNode
+	 *
+	 * @since 3.0
+	 *
+	 */
+	public function testGraphNode(){
+
+		//can create GraphNode
+		$this->assertInstanceOf(
+			GraphNode::class,
+			$node = new GraphNode( 'Team:Beta' )
+		);
+
+		$this->assertEquals( 'Team:Beta', $node->getID() );
+
+		$node->setLabel( "Fossil Power Generation" );
+		$this->assertEquals( "Fossil Power Generation", $node->getLabel() );
+
+		$mockParentNode1[] = [
+			"predicate" => 'Part Of Team',
+			"object"    => 'Alpha Team'
+		];
+
+		$node->addParentNode( 'Part Of Team', 'Alpha Team' );
+		$this->assertEquals( $mockParentNode1, $node->getParentNode() );
+
+	}
```

Comment:
Please follow http://wiki.c2.com/?ArrangeActAssert. You can create more than one test method.

Reply:
see: https://github.com/RacoonDev/SemanticResultFormats/commit/c1acdf0cf7f136a108ae0380959954703d0bbd35